addon/components/swiper-container.js

```diff
@@ -206,19 +204,36 @@ export default Component.extend({
    * @param {Object} swiper - Swiper instance
    */
   _slideChanged(swiper) {
-    let index = this.get('loop')
-      ? $(swiper.slides)
-        .filter('.swiper-slide-active')
-        .attr('data-swiper-slide-index')
-      : swiper.realIndex;
+    let index;
+
+    if (this.get('loop')) {
+      Object.keys(swiper.slides).forEach((key) => {
+        if (swiper.slides[key].classList && swiper.slides[key].classList.contains('swiper-slide-active')) {
```

Comment:
Why is this check for classList necessary?

Reply:
forEach is less efficient than filter.

Reply:
swiper.slides is an object like this
```javascript
 { 0: HTMLElement, 1: HTMLElement, length: 2}
```
So I was just ensuring it was an HTMLElement, otherwise it will crash.